### PR TITLE
Add new field to create relationship in additional options section

### PR DESCRIPTION
**Pull request by jitendrapurohit**
Fix for https://www.drupal.org/node/2511208

### CODE REVIEW

File: includes/wf_crm_webform_base.inc [hunk 1]
```diff
@@ -417,6 +417,10 @@ abstract class wf_crm_webform_base {
    */
   protected function getRelationship($r_types, $cid1, $cid2) {
     $found = array();
+    $active_clause = '';
+    if ($this->settings['create_new_relationship']) {
```
Comment (colemanw) [inline includes/wf_crm_webform_base.inc]: I think you should wrap this variable in `!empty()` to avoid a possible php notice if it's not defined.

**Review by colemanw — CHANGES_REQUESTED** [review]

File: includes/wf_crm_webform_base.inc (hunk 1 above)
Comment (jitendrapurohit) [reply includes/wf_crm_webform_base.inc]: Updated :-)

File: includes/wf_crm_webform_postprocess.inc [hunk 2]
```diff
@@ -966,6 +966,10 @@ class wf_crm_webform_postprocess extends wf_crm_webform_base {
           $perm = $perm == 1 ? 2 : 1;
         }
       }
+      //initialise start_date for create action.
+      if (empty($existing) !array_key_exists('start_date', $params) {
```
Comment (colemanw) [inline includes/wf_crm_webform_postprocess.inc]: I'm having trouble reading this if statement. There's no operator between the two conditions.
Also it looks like there's a missing closing bracket. Did you try running this code? Looks like you'd get a fatal syntax error.

**Review by colemanw — CHANGES_REQUESTED** [review]

File: includes/wf_crm_webform_postprocess.inc (hunk 2 above)
Comment (jitendrapurohit) [reply includes/wf_crm_webform_postprocess.inc]: seems like it got messed up while pushing, was working fine while testing. I've tested the flow again and confirmed that it creates a new relationship when an expired/inactive duplicate relationship exists for a contact.

Thanks for spotting!

**Pull request merged**